pike/test/compound.py

```diff
@@ -65,7 +67,7 @@ def __init__(self, tree=None):
 class CompoundTest(pike.test.PikeTest):
 
     # Compounded create/close of the same file, with maximal access request
-    def test_create_close(self):
+    def generic_test_create_close(self, use_invalid_sid=False, use_invalid_tid=False):
```

Comment:
it would be preferred to pass in the sid and tid to use as opposed to hiding this detail inside the function

Reply:
Er, okay.  Strangely, that's how I coded it first, but then decided to go this route instead.  Not sure I see one way being preferable to the other, but I'm fine with changing it.

Reply:
(switched to my isi account)

the test is more flexible if the sid and tid can be passed in -- if someone ever wanted to add tests that use a different, particular session id or something, then it lends itself to that.

at the end of the day, it doesn't really matter, it's test code. if someone needs to do something different, they will just change it.

Reply:
I decided you're probably right.  But then went to test my changes and decided I ought to run it with pike_pp_tester.py, just to be safe.  Sadly,  it croaks. :-(   (I hadn't tried that with my original version, but I'm sure it would have failed in the same way).  So, PR is on hold for a bit, since I don't want to introduce changes that will break our pp accounting testing.  :-}  I'm consulting with pfawcett.  So, more news when I have some!

Reply:
Btw, just to give you an update on the above.  pfawcett investigated and determined there was a bug in his pp_accounting code that was causing the problem.  So my thought is that we just hold off on doing this PR until he gets that fixed.  (I'm not sure what the timeframe for that will be though; it wasn't at the top of his priority list).